libs/libcommon/src/libcommon/simple_cache.py

```diff
@@ -127,6 +161,31 @@ class CacheEntryDoesNotExistError(DoesNotExist):
     pass
 
 
+class MultipleCacheError(MultipleObjectsReturned):
+    pass
+
+
+def update_metrics(kind: str, http_status: HTTPStatus, inc__total: int, error_code: Optional[str] = None) -> None:
```

Comment:
nit: why two `__` in `inc__total`?

Reply:
why not `increase_by`, maybe it's clearer

Reply:
hmmm ok it's mongoengine syntax. Even though: maybe use `increase_by` in the method signature, then pass it to `inc__total` in the mongoengine method

Reply:
Done in https://github.com/huggingface/datasets-server/pull/1658/commits/b6650cff6125c20e7e878500bb156de7384b1cf6